### DISCUSSION TITLE
Is there any way to make AvatarView clickable?

### DISCUSSION THREAD

**Discussion by nosferatu500** [CommunityToolkit/Maui#675] [Q&A] [answered]
I would like to use it something like this:
`<toolkit:AvatarView Text="Avatar" Command="{Binding GoToProfileCommand}" />`

**Comment by xDaijobu** [comment, marked as answer]
You can make an element tappable by using a [GestureRecognizer](https://learn.microsoft.com/en-us/dotnet/maui/fundamentals/gestures/tap)

```
<xct:AvatarView Text="Avatar">
    <xct:AvatarView.GestureRecognizers>
        <TapGestureRecognizer Command="{Binding GoToProfileCommand}"/>
    </xct:AvatarView.GestureRecognizers>
</xct:AvatarView>
```

**Reply by GeorgeLeithead** [reply]
There are also examples in the sample app for all the current gestures, some 60+ for AvatarView alone! Enjoy.  Let us know if you need more?

**Comment by CliffAgius** [comment]
Closed as answered
